code/hud/hudsquadmsg.cpp

```diff
@@ -1239,7 +1240,7 @@ int hud_squadmsg_send_ship_command( int shipnum, int command, int send_message,
 			return 0;
 		
 		case STAY_NEAR_ME_ITEM:
-		case STAY_NEAR_TARGET_ITEM:		// can't attack anything on same team
```

Comment:
Overall, also looks like there's multiple cases where comments are removed, but tagging all of those would likely be redundant at this early review stage before hearing back.

Reply:
I don't think this particular comment is relevant either